aspnetcore/blazor/hosting-models.md

```diff
@@ -153,8 +188,18 @@ For more information on Microsoft native client frameworks, see the following re
 
 ## Which Blazor hosting model should I choose?
 
+:::moniker range=">= aspnetcore-8.0"
+
+A component's hosting model is set by its *render mode*, either at compile time or runtime, which is described with examples in <xref:blazor/fundamentals/render-modes>. We recommend waiting to read the *Render modes* article until after you've read the articles leading up to it. The following table shows the primary considerations for setting the render mode to determine a component's hosting model. For static Blazor WebAssembly apps, all of the app's components are rendered on the client with the Blazor WebAssembly hosting model.
```

Comment:
```suggestion
A component's hosting model is set by its *render mode*, either at compile time or runtime, which is described with examples in <xref:blazor/fundamentals/render-modes>. We recommend waiting to read the *Render modes* article until after you've read the articles leading up to it. The following table shows the primary considerations for setting the render mode to determine a component's hosting model. For standalone Blazor WebAssembly apps, all of the app's components are rendered on the client with the Blazor WebAssembly hosting model.
```

Reply:
I'll sweep the whole doc set for this change before merging this.